deployment/hasura/migrations/AerieMerlin/22_constraint_violation_schema/up.sql

```diff
@@ -0,0 +1,6 @@
+-- The schema of the database is unchanged, but the JSON schema of the constraint violations in constraint_run is different.
+-- Theoretically we could use SQL to transform the schemas of existing JSON objects, but I'd rather just delete them.
+-- programming is hard, ok
```

Comment:
Minor: comment tweak proposed for clarity/accuracy

```suggestion
-- Theoretically we could use SQL to transform the schemas of existing JSON objects, but since the table is just a cache it is safe to clear it
```

Reply:
<sub>do you mean to suggest that programming is easy</sub>

Reply:
<sub>oh sorry let me fix that suggestion</sub>
```suggestion
-- Theoretically we could use SQL to transform the schemas of existing JSON objects, but since the table is just a cache it is safe to clear it
-- programming is hard, ok
```